src/views/PostPage.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="page">
+    <h1 class="entry-title">ARTICLES</h1>
+    <div class="content" v-html="content">
+      <Articles v-for="post in posts" :key="post.id" :post="post" />
```

Comment:
Same as for Selections - the 'post'-variable doesn't exist in this component. All the posts are fetched in <Articles>. So this line can just be:
`<Articles />`